app/plugin_test.go

```diff
@@ -312,3 +315,51 @@ func TestGetPluginStatuses(t *testing.T) {
 	require.Nil(t, err)
 	require.NotNil(t, pluginStatuses)
 }
+
+func TestPluginSync(t *testing.T) {
```

Comment:
Maybe want to write run this test with the S3 interface as well?

Reply:
good call!

Reply:
updated.

Reply:
Great!